### DISCUSSION TITLE
Whoops \ Exception \ ErrorException (E_ERROR)

### DISCUSSION THREAD

**Discussion by RaykDev01** [DirectoryLister/DirectoryLister#1123] [Help & Support] [answered]
when i use the search bar for find anything i receive this error:

 Whoops \ Exception \ ErrorException (E_ERROR)
Maximum execution time of 30 seconds exceeded

please help. this is my only issue with this template. thanks 

i have linux mint the last version and php v8

**Comment by PHLAK** [comment, marked as answer]
You need to [increase the `max_execution_time` in your `php.ini` file](https://code.tutsplus.com/how-to-increase-max_execution_time-in-php--cms-37017t).

**Reply by RaykDev01** [reply]
Work now but.... take three minutes for search done. Any action for make? In you example the searching is fast. Thank you
